entities/search/result.go

```diff
@@ -71,55 +71,55 @@ func (r Result) Thing() *models.Thing {
 	return t
 }
 
-func (r Result) Action() *models.Action {
-	schema, ok := r.Schema.(map[string]interface{})
-	if ok {
-		delete(schema, "uuid")
-	}
+// func (r Result) Action() *models.Action {
```

Comment:
There is quite some commented code left, also in the `objects` package. Would be nice to eventually delete all of this, but I'm fine with scoping this out to another issue to save time right now as well.

Reply:
I'll remove this